### ISSUE TITLE
The space disappears after converting complex value

### ISSUE THREAD

**Issue by sgromkov** [Lakkanna/css-converter#22]
Original CSS:
```css
border-radius: 100%/130% 130% 15px 15px;
box-shadow: 0 1px 2px rgba(0,0,0,.5);
text-shadow: -1px -1px 0 #000,2px -1px 0 #000,2px 2px 0 #000,-1px 2px 0 #000,2px 3px 0 #000,1px 3px 0 #000,2px 3px 0 #000;
```

❌ Current conversion result:
```jsx
borderRadius: "100%/130%130%15px15px",
boxShadow: "01px2pxrgba(0,0,0,.5)",
textShadow: "-1px-1px0#000,2px-1px0#000,2px2px0#000,-1px2px0#000,2px3px0#000,1px3px0#000,2px3px0#000",
```

✅ Expected result:
```jsx
borderRadius: "100%/130% 130% 15px 15px",
boxShadow: "0 1px 2px rgba(0,0,0,.5)",
textShadow: "-1px -1px 0 #000, 2px- 1px 0 #000, 2px 2px 0 #000, -1px 2px 0 #000, 2px 3px 0 #000, 1px 3px 0 #000, 2px 3px 0 #000",
```

**Comment by Lakkanna** [comment]
Hi @sgromkov
I'm sorry for the delayed response on this issue, The issue has been fixed in version 0.2.0 now

Current conversion result
```
borderRadius: "100%130% 130% 15px 15px",
boxShadow: "0 1px 2px rgba(0,0,0,.5)",
textShadow: "-1px -1px 0 #000,2px -1px 0 #000,2px 2px 0 #000,-1px 2px 0 #000,2px 3px 0 #000,1px 3px 0 #000,2px 3px 0 #000",
```

There is still an issue related to providing a space after a comma (,). We will fix that soon.

**Comment by Lakkanna** [comment]
Hi @sgromkov,

I'm glad to inform you that the issue you reported has been fixed in the latest version. We've closed this issue for now, but if you find that the issue still persists in the latest version, please feel free to reopen this issue and provide us with more details.

Thank you for bringing this issue to our attention and helping us to improve extension.
